docs/products/mysql/howto/max-number-of-connections.rst

```diff
@@ -0,0 +1,29 @@
+Calculate the maximum number of connections for MySQL
+=====================================================
+
+
+The number of simultaneous connections in Aiven for MySQL depends on the usable memory on the server. The usable memory is the total memory on the node minus the operating system and management overhead. This overhead is currently estimated as 350 MiB (0.341797 GiB). 
+
+It can be estimated as:
+
+.. math::
+
+    {usable\_memory} = {total\_memory\_on\_the\_node - management\_overhead} 
+
+.. note::
+    
+
+To calculate the maximum number of connections, we need to check the value of the usable memory. The usable memory is rounded to the nearest GiB. For plans under 4 GiB of usable memory, you can estimate as:
```

Comment:
I think this should explicitly say `rounded down` (which is what the *help* article said)

Reply:
The examples make this nice and clear - thank you.